powerspikegg/computation_models/match/freeze.sh

```diff
@@ -0,0 +1,9 @@
+#!/bin/bash
+
+stats=(kills deaths assists minions_killed neutral_minions_killed total_damages total_heal wards_placed tower_kills) 
```

Comment:
You should overquote those names (i.e. `("kills" "deaths" "assists" "minions_killed" "neutral_minions_killed" "total_damages" "total_heal" "wards_placed" "tower_kills")`)